_docs_integrate/succeeding-attributes-to-update-their-values.md

```diff
@@ -46,7 +46,7 @@ When [creating an IdentityAttribute]({% link _docs_integrate/create-attribute-fo
 Instead, it will be stored in the `content` property of a LocalAttribute with an undefined `shareInfo`.
 We refer to these unshared LocalAttributes as RepositoryAttributes, since they make up your private repository of Attributes.
 
-In the following examples, the `createdAt` property all occuring LocalAttribute is omitted, since it isn't required for the explanation.
+In the following examples, the `createdAt` and `deletionInfo` properties of all occuring LocalAttribute is omitted, since they aren't required for the explanation.
```

Comment:
- "LocalAttributes" instead of "LocalAttribute"
- The concepts of Attribute succession and Attribute deletion nevertheless overlap. I'd reformulate like that: 

"In the following examples, the `createdAt` property of all occuring LocalAttributes is omitted, since it isn't required for the explanation. Furthermore, as it is not relevant for gaining a basic understanding of the concept of Attribute succession, it is assumed in the following that the deletion of Attributes does not come into play and, accordingly, the `deletionInfo` property of all occuring LocalAttributes in particular is also omitted. How the succession of Attributes and the deletion of Attributes impact each other is explained in the documentation of the [Delete an Attribute]/[Deletion of Attributes] scenario."

Reply:
That's right, you don't need the Attribute deletion to understand the Attribute succession, so I can understand why the sentence here may should be kept simple. However, if this is done, I'd be in favor of pointing out the Attribute deletion in a 'What's next?' section or similar, as it is important to make connections between the different scenarios through links. Especially if the concepts overlap with each other, as is the case with the Attribute succession and Attribute deletion.